WordPress/src/main/res/layout/login_magic_link_request_screen.xml

```diff
@@ -0,0 +1,48 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Thanks for tip! I actually try to stay away from following AS reformat on the layouts until we have a common "codestyle" in place. I know it's annoying but, yeah, hope we'll make it happen in the future :).